README.md

```diff
@@ -1,6 +1,6 @@
 # SerenityOS
 
-Graphical Unix-like operating system for x86 computers.
+Graphical Unix-like operating system for x86 and x86_64 computers.
```

Comment:
This change is not necessary. "x86 computers" includes both 32-bit and 64-bit CPUs.